tests/components/zha/test_cluster_handlers.py

```diff
@@ -980,3 +982,19 @@ async def func(arg1: int, arg2: int) -> int:
     assert func.await_count == 3
     assert isinstance(exc.value, HomeAssistantError)
     assert str(exc.value) == expected_error
+
+
+async def test_cluster_handler_naming() -> None:
+    """Test that all cluster handlers are named appropriately."""
+    for client_cluster_handler in registries.CLIENT_CLUSTER_HANDLER_REGISTRY.values():
+        assert issubclass(client_cluster_handler, cluster_handlers.ClientClusterHandler)
+        assert client_cluster_handler.__name__.endswith("ClientClusterHandler")
+
+    for cluster_handler_dict in registries.ZIGBEE_CLUSTER_HANDLER_REGISTRY.values():
+        for cluster_handler in cluster_handler_dict.values():
+            # this cluster handler needs to be looked at closer. Will do in Update platform PR
+            if cluster_handler.__name__ != "OtaClientClusterHandler":
```

Comment:
Just remembered this discussion about treating a special case in the test itself: https://github.com/home-assistant/core/pull/102806#discussion_r1372364965
But in this case, it would only be temporary (until changed in your OTA PR).